semantic/src/site/globals/site.overrides

```diff
@@ -1290,7 +1306,9 @@ a:hover {
 #app-navigation-content {
   .flex-row();
   flex-wrap: wrap;
-  width: @applicationFormContentWidth;
+  max-width: @applicationFormContentWidth;
+  justify-content: space-between;
+  gap: 10px;
```

Comment:
Not directly related to issue, but this just makes the Previous/Next and Submit navigation at the bottom wrap a bit nicer when the viewport gets tight.